core/src/components/datetime/usage/vue.md

```diff
@@ -133,10 +136,20 @@
         return format(parseISO(value), 'MMM dd yyyy');
       };
 
+      const isDateEnabled = (dateIsoString: string) => {
+        const date = new Date(dateIsoString);
+        if (date.getUTCDay() === 1 && date.getUTCMonth() === 0 && date.getUTCFullYear() === 2022) {
```

Comment:
getUTCDay gives the day of the week. I think you want getUTCDate which gives the calendar day of the month.

Reply:
Non-blocking, but we typically push users to use date-fns for any date parsing. It would be nice to show some date-fns usage here as well.